libs/telemetry/src/registry.cpp

```diff
@@ -90,6 +90,10 @@ CounterPtr Registry::CreateCounter(std::string name, std::string description, La
       measurements_.push_back(counter);
     }
   }
+  else
+  {
+    std::cout << LOGGING_NAME << ": Failed to validate telemetry name: " << name << std::endl;
```

Comment:
Not 100% keen on having multiple things talking to `cout` at the same time. Was this just debug?